ci/travis/format.sh

```diff
@@ -121,20 +193,49 @@ format_changed() {
                  clang-format -i
         fi
     fi
+
+    if command -v shellcheck >/dev/null; then
+        if ! git diff --diff-filter=ACRM --quiet --exit-code "$MERGEBASE" -- 'WORKSPACE' 'WORKSPACE.*' 'BUILD.*' '*.bzl' '*.bazel' &>/dev/null; then
+            shellcheck_bazel
+        fi
+
+        local shell_files
+        # shellcheck disable=SC2207
+        shell_files=($(
+          git diff --name-only --diff-filter=ACRM "$MERGEBASE" -- '*.sh' &&
+          git diff --name-only --diff-filter=ACRM "$MERGEBASE" -- ':(exclude)*.*' | xargs -r git --no-pager grep -l '^#!\(/usr\)\?/bin/\(env \+\)\?\(ba\)\?sh'
+        ))
+        if [ 0 -lt "${#shell_files[@]}" ]; then
+            shellcheck_scripts "${shell_files[@]}"
+        fi
+    fi
 }
 
 # Format all files, and print the diff to stdout for travis.
 format_all() {
+    flake8 --inline-quotes '"' --no-avoid-escape --exclude=python/ray/core/generated/,streaming/python/generated,doc/source/conf.py,python/ray/cloudpickle/,python/ray/thirdparty_files/ --ignore=C408,E121,E123,E126,E226,E24,E704,W503,W504,W605
```

Comment:
can we set up a flake8 config instead of copying these ignore flags in many different places?

Reply:
Possibly. Note that this isn't new logic here though -- I just moved it from the other file. I have no idea how to set up a flake8 config but it might be worth doing in a separate PR.